src/main/resources/templates/macros/report/element.vm

```diff
@@ -9,12 +9,13 @@
 
   #includeTags($element.getTags())
 
-  <span data-toggle="collapse" class="#if ($element.getElementStatus().isPassed()) collapsed #end collapsable-control" data-target="#element-$element.hashCode()">
+  #set($elementId = $counter.getNext())
+  <span data-toggle="collapse" class="#if ($element.getElementStatus().isPassed()) collapsed #end collapsable-control" data-target="#element-$elementId">
     #includeBrief($element.getKeyword(), $element.getElementStatus(), $element.getName(), true)
   </span>
   <div class="description indention">$element.getDescription()</div>
 
-  <div id="element-$element.hashCode()" class="collapse collapsable-details #if(!$element.getElementStatus().isPassed()) in #end">
+  <div id="element-$elementId" class="collapse collapsable-details #if(!$element.getElementStatus().isPassed()) in #end">
```

Comment:
ok - anyway I don't see a problem when the counter is continued between pages, so this is global

it must be unique per page so when is unique per whole report it's fine as well, isn't it?